src/blog/wasm-decompile.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "What's in that .wasm? Introducing: wasm-decompile"
+author: 'Wouter van Oortmerssen ([@wvo](https://twitter.com/wvo))'
+avatars:
+  - 'wouter-van-oortmerssen'
+date: 2020-04-20 12:35:00
+tags:
+  - WebAssembly
+  - tooling
+description: 'WABT gains a new decompilation tool that can make it easier to
+read the contents of Wasm modules'
+---
+We have a growing number of compilers and other tools that generate or
```

Comment:
Throughout the file: don't use hard line breaks unless you semantically want to insert a hard break.

Reply:
See discussion above.. this is not how anyone else uses markdown. But whatever.